tests/ignite/contrib/metrics/test_average_precision.py

```diff
@@ -63,21 +63,21 @@ def test_check_shape():
         ap._check_shape((torch.rand(4, 3), torch.rand(4, 3, 1)))
 
 
-def test_binary_input_N():
+def test_binary_input():
```

Comment:
I think the name is not well chosen. We have now binary and multilabel test cases in this test.
Let's rename it.